node/service/src/service.rs

```diff
@@ -262,9 +266,39 @@ async fn start_node_impl(
 		})
 		.await?;
 
+	// Start off-chain workers if enabled
+	if parachain_config.offchain_worker.enabled {
+		log::info!("OFFCHAIN WORKER is Enabled!");
+		let offchain_workers =
+			sc_offchain::OffchainWorkers::new(sc_offchain::OffchainWorkerOptions {
+				runtime_api_provider: client.clone(),
+				is_validator: validator,
```

Comment:
I see the offchain workers have access to runtime API via the client, we can use that to do any runtime call ?

Reply:
since offchain worker is part of the wasm it can access the runtime function but to apply changes to on-chain data it should call one of the following functions
```
Signed Transactions
Unsigned Transactions w/ Signed Payloads
Unsigned Transactions
```